prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.ts

```diff
@@ -82,13 +87,21 @@ export class OverviewComponent implements OnInit {
   }
 
   public nextRoute(): void {
-    this.routeUtils.routeTo([SiteRoutes.MODULE_PATH, SiteRoutes.SITE_MANAGEMENT], {
-      queryParams: { submitted: true }
-    });
+    this.busy = this.determineOrgAgreementRequired$().subscribe(
+      ignored => {
```

Comment:
If you want to ignore a function parameter its convention in JavaScript or TypeScript that you use an underscore - `(_) => { }`